static/third/jquery-filedrop/jquery.filedrop.js

```diff
@@ -95,7 +95,7 @@
         global_progress = []
         // Zulip modification: keep a pointer to the object that the function
         // was invoked on.
-        caller = this;
+    var caller = this;
```

Comment:
The actual bug here is a missing comma on the previous line.

Reply:
Regardless, it seems like a better strategy to only declare one variable per statement, especially in our changes. Since its an external library, we shouldn't make that change to their code, but we should ensure we follow our version of 'good practices' here.

Reply:
You’re not wrong, but in that case, we should add back the semicolon that we removed in 932e98f4568e95805953a25a22a7e0886caf48c6, and fix the misleading comment indentation.